### PR TITLE
Fix address in etherscan for PolygonZkEvmBridge.sol

### PR DESCRIPTION
**Pull request by omnifient**
Previous address is for PolygonZkEVMUpgraded.sol, not the bridge

### CODE REVIEW

File: packages/config/src/layer2s/polygonzkevm.ts [hunk 1]
```diff
@@ -408,7 +408,7 @@ export const polygonzkevm: Layer2 = {
         references: [
           {
             text: 'PolygonZkEvmBridge.sol - Etherscan source code, claimAsset function',
-            href: 'https://etherscan.io/address/0xb1585916487AcEdD99952086f2950763D253b923',
+            href: 'https://etherscan.io/address/0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/polygonzkevm.ts:411]:
```suggestion
            href: 'https://etherscan.io/address/0x0feb850b183c57534b56b7d56520133c8f9bdb65',
```

File: packages/config/src/layer2s/polygonzkevm.ts [hunk 2]
```diff
@@ -408,7 +408,7 @@ export const polygonzkevm: Layer2 = {
         references: [
           {
             text: 'PolygonZkEvmBridge.sol - Etherscan source code, claimAsset function',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/polygonzkevm.ts:410]:
```suggestion
            text: 'PolygonZkEvmBridgeV2.sol - Etherscan source code, claimAsset function',
```

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/polygonzkevm.ts (hunk 1 above)
Comment (omnifient) [reply packages/config/src/layer2s/polygonzkevm.ts:411]: huh, bridging/claiming is done through the proxy (0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe), not the implementation (0x0feb850b183c57534b56b7d56520133c8f9bdb65), so why point to the impl?

Comment (lucadonnoh) [reply packages/config/src/layer2s/polygonzkevm.ts:411]: because we want to show the line of code that implements the functionality

**Pull request merged**